UPGRADE-14.0.md

```diff
@@ -113,6 +113,18 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
     -   see #project-base-diff to update your project
 -   replace custom application bootstrapping with symfony/runtime ([#2914](https://github.com/shopsys/shopsys/pull/2914))
     -   see #project-base-diff to update your project
+-   move productsByCatnums endpoint from project-base to frontend-api package ([#2716](https://github.com/shopsys/shopsys/pull/2716))
+    - constructor of `Shopsys\FrontendApiBundle\Model\Resolver\Products\ProductsQuery` has cahnged:
+    ```diff
+        public function __construct(
+            protected readonly ProductFacade $productFacade,
+            protected readonly ProductFilterFacade $productFilterFacade,
+            protected readonly ProductConnectionFactory $productConnectionFactory,
+    +       protected readonly DataLoaderInterface $productsVisibleAndSortedByIdsBatchLoader,
+    +       protected readonly ProductRepository $productRepository,
+        )
+    ```
+    - see #project-base-diff to update your project
```

Comment:
There are multiple changes in framework, that are not reflected here.

Reply:
This is very confusing - if you have cherry-picked it here and it will be committed with this change, I think it should be added here, as someone can update to this change only.

Reply:
ok, makes sense, after refactoring the logic, there was just one thing left for mentioning in the upgrade notes, added in 58a387fe2